lib/bin/repl.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2024 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const should = require('should'); // eslint-disable-line import/no-extraneous-dependencies
+require('../../test/assertions');
+
+const { sql } = require('slonik');
+
+const container = require('../util/default-container');
+
+(async () => {
+  const context = { ...container, should, sql };
```

Comment:
Do you think we could remove `with` from `context`? It's somehow surprising to see it in the list, and I doubt we'd want to call it directly (i.e., running `with()`), since it's usually run as `container.with()`. As long as we add `container` to `context`, then it'll be possible to call `container.with()`. I'm not sure it'd even be possible to call `with()` directly — I don't think `with` is a valid identifier.